include/ensmallen_bits/pso/update_policies/lbest_update.hpp

```diff
@@ -20,12 +17,44 @@
 namespace ens {
 
 /**
- * ADD SLIGHTLY MORE DETAILED DESCRIPTION HERE.
+ * The local best version (lbest) of PSO in which particles communicate with
+ * only two neighbours each, thus forming a ring topology amongst them. This
+ * approach allows PSO to converge at the global minimum, but takes
+ * significantly more iterations to do so.
+ *
+ * The lbest update scheme is described as follows:
+ *
+ * \f{eqation}{
+ * v_{i+1} = \phi (v_i + c_1 * r_1 * (p_{best} - p_{current}) +
+ *           c_1 * r_1 * (l_{best} - p_{current}))
+ * \f}
  *
- * For more information, see the following.
+ * where \f$ v_i \f$ is the velocity of a particle in iteration \f$ i \f$,
+ *       \f$ p_{best} \f$ is the best position of an individual particle,
+ *       \f$ p_{current} \f$ is the current position of the particle,
+ *       \f$ l_{best} \f$ is the local best position,
+ *       \f$ r_1 \f$ and \f$ r_2 \f$  are standard uniform random variables,
+ *       \f$ c_1 \f$ is the exploitation factor,
+ *       \f$ c_2 \f$ is the exploration factor, and
+ *       \f$ \phi \f$ is the constriction factor.
  *
- * REFERENCE TO A RELEVANT ARTICLE.
+ * For more information, refer the following:
  *
+ * @code
+ * @article{Poli2007,
+ *          doi = {10.1007/s11721-007-0002-0},
+ *          url = {https://doi.org/10.1007/s11721-007-0002-0},
+ *          year = {2007},
+ *          month = aug,
+ *          publisher = {Springer Science and Business Media {LLC}},
```

Comment:
That company is typically referred to as "Springer", not as "Springer Science and Business Media {LLC}".  
It's actually better to leave out the publisher field as it's not necessary.

Reply:
Thank you for pointing this out! I'll make the changes.